dashboard/modules/@apostrophecms/passport-bridge/index.js

```diff
@@ -0,0 +1,79 @@
+import * as client from 'openid-client';
+import {
+  Strategy
+} from 'openid-client/passport'
+
+export default {
+  options: {
+    create: {
+      role: 'guest'
+    },
+    strategies: [
+      {
+        async factory(params, fn) {
+          const issuer = new URL(process.env.OIDC_ISSUER);
```

Comment:
Because of discovery, all we need is a URL to the provider site (OIDC_ISSUER), OIDC_CLIENT_ID, and OIDC_CLIENT_SECRET.